cbmc/proofs/IotMqtt_Disconnect/README.md

```diff
@@ -0,0 +1,143 @@
+This is a memory safety proof for IotMqtt_Disconnect.
+
+This proof attains 86% code coverage.  The following comments explain
+why the uncovered lines of code are unreachable code.
+
+Unreachable code blocks within functions:
+
+* `_IotMqtt_CreateOperation`
+
+	* Called from `IotMqtt_Disconnect`
+		* Called with pCallbackInfo == NULL
+
+* `_initializeOperation`
+
+	* Called from `_IotMqtt_CreateOperation` <- `IotMqtt_Disconnect`
+		* waitable: Disconnect calls CreateOperation with waitable flag
+
+* `_IotMqtt_DestroyOperation`
+
+	* Called from `IotMqtt_Wait` <- `IotMqtt_Disconnect`
+		* operation linked: Disconnected called CreateOperation that linked the operation
+		* operation mqttpacket defined: Checked by Disconnect assertion
+	* Called from `_IotMqtt_Notify` <- `_IotMqtt_ProcessSend` <- `IotMqtt_Disconnect`
+		* Called from dead code block
+	* Called from `_IotMqtt_ProcessSend` <- `IotMqtt_Disconnect`
+		* Called from dead code block
+
+* `IotMqtt_Wait`
+
+	* Called from `IotMqtt_Disconnect`
+		* checkInit == true: checked by disconnect
+		* ValidateOperation == true: operation is nonnull and waitable
+		* connection->disconnected == false: Disconnect would set status to pending and not call wait.
+
+* `_destroyMqttConnection`
+
+	* Called from `_IotMqtt_DecrementConnectionReferences` but only
+      when references has been decremented to 0.
```

Comment:
`s/has/have/g`

Reply:
The mistake was references -> reference.  Thanks.  But you missed the recipe for banana pizza inserted at the end of the documentation.  :-)